client/landing/gutenboarding/onboarding-block/edit.tsx

```diff
@@ -21,110 +22,142 @@ const siteTypeOptions: Record< SiteType, string > = {
+	const { setSiteType, setSiteTitle } = useDispatch( STORE_KEY );
 
 	const updateTitle = useCallback(
 		( e: React.ChangeEvent< HTMLInputElement > ) => setSiteTitle( e.target.value ),
 		[ setSiteTitle ]
 	);
+
 	const updateSiteType = useCallback(
 		( e: React.ChangeEvent< HTMLInputElement > ) => setSiteType( e.target.value as SiteType ),
 		[ setSiteType ]
 	);
 
+	const selectSiteType = e => {
+		updateSiteType( e );
+		setLastSelectedQuestion( null );
+	};
+
+	const openSiteType = () => {
+		setLastSelectedQuestion( 'siteType' );
+	};
+
+	const selectSiteVertical = () => {
+		setLastSelectedQuestion( 'siteVertical' );
+	};
+
+	const selectSiteTitle = () => {
+		setLastSelectedQuestion( 'siteTitle' );
+	};
+
+	const isSelected = questionType => {
+		if ( lastSelectedQuestion ) {
+			return questionType === lastSelectedQuestion;
+		} else if ( isEmpty( siteType ) ) {
+			return questionType === 'siteType';
+		} else if ( isEmpty( siteVertical ) ) {
+			return questionType === 'siteVertical';
+		} else if ( questionType === 'siteTitle' ) {
+			return true;
+		}
+		return false;
+	};
+
 	return (
-		<div className="onboarding-block__questions">
-			<h2 className="onboarding-block__questions-heading">
-				{ NO__( "Let's set up your website – it takes only a moment." ) }
-			</h2>
-
-			<div className="onboarding-block__question">
-				<span>{ NO__( 'I want to create a website ' ) }</span>
-				{ ! isFilledFormValue( siteType ) ? (
-					<ul className="onboarding-block__multi-question">
-						{ map( siteTypeOptions, ( label, value ) => (
-							<li key={ value }>
-								<label>
-									<input
-										name="onboarding_site_type"
-										onChange={ updateSiteType }
-										type="radio"
-										value={ value }
-									/>
-									<span className="onboarding-block__multi-question-choice">{ label }</span>
-								</label>
-							</li>
-						) ) }
-					</ul>
-				) : (
-					<div className="onboarding-block__multi-question">
-						<button className="onboarding-block__question-answered" onClick={ resetSiteType }>
-							{ siteTypeOptions[ siteType ] }
-						</button>
+		<div className="onboarding-block__acquire-intent">
+			<div className="onboarding-block__questions">
+				<h2 className="onboarding-block__questions-heading">
+					{ isEmpty( siteType ) && NO__( "Let's set up your website – it takes only a moment." ) }
+				</h2>
+
+				<div
+					className={ classNames( {
+						'onboarding-block__question': true,
+						selected: isSelected( 'siteType' ),
+					} ) }
+				>
+					<span>
+						{ isEmpty( siteType )
+							? NO__( 'I want to create a website ' )
+							: NO__( 'Setting up a website ' ) }
+					</span>
+					{ isSelected( 'siteType' ) ? (
+						<ul className="onboarding-block__multi-question">
+							{ map( siteTypeOptions, ( label, value ) => (
+								<li key={ value } className={ value === siteType ? 'selected' : null }>
```

Comment:
Nit, we could make things more semantic if we used a CSS selector to style the label for a selected radio button, see https://stackoverflow.com/questions/4641752/css-how-to-style-a-selected-radio-buttons-label. This would allow us to drop the change on this line :slightly_smiling_face: